test/docfx.E2E.Tests/DocfxSeedSiteTest.cs

```diff
@@ -114,8 +115,9 @@ public void TestReferencePage()
 
             // check overwrite
             var conceptual = _driver.FindElement(By.ClassName("conceptual"));
-            // Add this line to help find out what sometime breaks e2e
-            Assert.True(conceptual.Text.Contains("This is a class talking about CAT."), $"Actual HTML: {conceptual.GetAttribute("outerHTML")}");
+            // Add these lines to help find out what sometime breaks e2e
```

Comment:
yes you're right, to could be missed

---
In reply to: [92092186](https://github.com/dotnet/docfx/pull/997#discussion_r92092186) [](ancestors = 92092186)